index.html

```diff
@@ -271,5 +271,59 @@ <h2 class="categorie-title">
     </ul>
 </section>
 </main>
+<footer class="footer-container" id="footer-container-id">
+  <div class="footer-form">
+    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    <form action="https://formspree.io/f/xzbovlzk" method="post">
+      <ul>
+        <li>
+          <input type="text" name="Name" placeholder="Full name">
+        </li>
+        <li>
+          <input type="email" name="Email" placeholder="Email address">
+        </li>
+        <li>
+          <textarea name="Message" id="msg" cols="30" rows="10">Enter text here</textarea>
+        </li>
```

Comment:
- [ ] Please, according to the requirement, All input fields should have the `required` attribute to activate the validation on submit, which means, the user can only submit the form after filling all input fields, kindly make sure to add this.